tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/service/impl/PlacementServiceImpl.java

```diff
@@ -799,6 +800,16 @@ public PlacementDTO closePlacement(final Long placementId) {
     return resultList;
   }
 
+  protected void populateEsrEventsForPlacementDetail(PlacementDetailsDTO placementDetailsDto) {
+    Set<PlacementEsrEvent> esrEvents = placementEsrEventRepository.
+        findPlacementEsrEventByPlacementIdIn(Collections.singletonList(
+            placementDetailsDto.getId()));
+    Set<PlacementEsrEventDto> esrEventDtos = new HashSet<>();
+    esrEvents.forEach(e -> esrEventDtos.add(
+        placementEsrEventDtoMapper.placementEsrEvenToPlacementEsrEventDto(e)));
+    placementDetailsDto.setEsrEvents(esrEventDtos);
```

Comment:
There are now a couple of places where this is done so I think this would be simpler if you follow the pattern of other mappers and define a `Set<?> placementEsrEventsToPlacementEsrEventDtos(Set<?>)`

Reply:
Fixed, thanks for pointing it out.